packages/admin-stories/.storybook/preview.tsx

```diff
@@ -3,7 +3,10 @@ import { addDecorator, addParameters } from "@storybook/react";
 import "@comet/admin-color-picker/src/themeAugmentation";
 import * as React from "react";
 import { IntlProvider } from "react-intl";
-import { createMuiTheme, MuiThemeProvider as ThemeProvider } from "@comet/admin";
+import { MuiThemeProvider as ThemeProvider } from "@comet/admin";
```

Comment:
should we move the `ThemeProvider` from `@comet/admin` to `@comet/admin-theme`

It's incompatible but i think the `ThemeProvider` makes most sense in a `@comet/admin-theme` package.

If we go further with `getThemeOptions` we could also remove `createMuiTheme` from `@comet/admin` and only work with `getThemeOptions` from `@comet/admin-theme`

Reply:
@manuelblum please take a look at what those two actually do https://github.com/vivid-planet/comet-admin/blob/next/packages/admin/src/mui/ThemeProvider.tsx

- ThemeProvider: wraps Mui ThemeProvider *plus* StyledComponents ThemeProvider, so we can work with both
- createMuiTheme: adds default values for rte theming values